procedures/katello/release.md.erb

```diff
@@ -32,7 +32,7 @@
 <% unless is_rc && !full_version.end_with?('1') -%>
   - Add `topics/katello-<%= version %>.adoc` to `guides/doc-Release_Notes/master.adoc`: `sed '/x.y.z releases here/a include::topics/katello-<%= version %>.adoc[leveloffset=+1]' master.adoc`
 <% end -%>
-  - Make sure [katello-contributors.adoc](https://github.com/theforeman/foreman-documentation/blob/<%= foreman_version %>/guides/doc-Release_Notes/topics/katello-contributors.adoc) is updated
+  - Update [katello-contributors.adoc](https://github.com/theforeman/foreman-documentation/blob/<%= foreman_version %>/guides/doc-Release_Notes/topics/katello-contributors.adoc) with the list of contributors using `git log --since="<YYYY-MM-DD of previous major release branching>" --format="%an" | sort | uniq`. Manually clean up any erroneous entries.
```

Comment:
Any chance we could standardize things by using the same script or reuse as much as possible?